matchMinima.py

```diff
@@ -432,13 +395,15 @@ def getRatioTimes(allMolTimes, zeroes):
 
     """
 
+#M Unrelated: You're compring times spent in each minima? What for?
```

Comment:
Ok so you're interested in the time it takes for optimization as well as the energy differences between levels of theory.